### PR TITLE
Start map

### PR DESCRIPTION
**Pull request by Alexander-Schulte**
##Node:
For now it only works if you add the court two times, the code for the coordinates is always one submit behind

Hello again! :) In this PR I add the the Geocoding API from Mapbox to my AddCourtPage. If the user is typing in an location and press on the Submit button, the Url will be fetched and generates the latitude and longitude of the given location. With this function, the added court gets the coordinates dynamically and not hardcoded by myself.
To check the function you can add a new court and add location (for now just real locations work) and check the local storage: the should be the "latitude" and the "longitude" with the exact coordinates! :) Next step is to use this coordinates to generate an icon on the mapbox map forEach added Court! :)
Thank u for your Review! :)

### CODE REVIEW

File: src/components/CourtCards.js [hunk 1]
```diff
@@ -42,11 +44,27 @@ function CourtCards() {
...
 
   function handleSubmit(event) {
+    function handleCourtCoordinates() {
+      const url = `https://api.mapbox.com/geocoding/v5/mapbox.places/${courtLocation}.json?access_token=pk.eyJ1Ijoic2VhcmNoLW1hY2hpbmUtdXNlci0xIiwiYSI6ImNrN2Y1Nmp4YjB3aG4zZ253YnJoY21kbzkifQ.JM5ZeqwEEm-Tonrk5wOOMw&cachebuster=1620130131393&autocomplete=true&country=de`;
+      console.log(courtLocation);
+      fetch(url)
+        .then((res) => res.json())
+        .then((apiData) => {
+          console.log(String(apiData.features[0].center[0]));
+          console.log(String(apiData.features[0].center[1]));
```
Comment (FriederikeM) [inline src/components/CourtCards.js:54]: no console logs allowed, mate

File: src/components/CourtCards.js [hunk 2]
```diff
@@ -42,11 +44,27 @@ function CourtCards() {
   }
 
   function handleSubmit(event) {
+    function handleCourtCoordinates() {
+      const url = `https://api.mapbox.com/geocoding/v5/mapbox.places/${courtLocation}.json?access_token=pk.eyJ1Ijoic2VhcmNoLW1hY2hpbmUtdXNlci0xIiwiYSI6ImNrN2Y1Nmp4YjB3aG4zZ253YnJoY21kbzkifQ.JM5ZeqwEEm-Tonrk5wOOMw&cachebuster=1620130131393&autocomplete=true&country=de`;
+      console.log(courtLocation);
```
Comment (FriederikeM) [inline src/components/CourtCards.js:49]: no no

**Review by FriederikeM — CHANGES_REQUESTED** [review]
Hey Alex, it looks pretty straight forward. However, when i enter an address  and check local storage, longitude and latitude are empty strings. Is there a specific way of entering the address that is required for it to work? If so, maybe notify the user about it somewhere

Screenshot 2021-05-04 at 20 16 20

On the map page i only get a blank screen, but not sure if the map is supposed to be rendering at this point 
Screenshot 2021-05-04 at 20 09 36

File: src/components/CourtCards.js (hunk 1 above)
Comment (Alexander-Schulte) [reply src/components/CourtCards.js:54]: Thank u! :) I removed all all the consol.logs, forgot that haha. mhhh weird.. have to check it again why the coordinates are empty now.

**Pull request merged**